src/assets/styles/generic/_vuetify.scss

```diff
@@ -39,6 +39,28 @@ html {
   border-radius: 2px;
 }
 
+.v-theme--light {
+  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat),  {
+    background-color: map-get($adm-colors, 'secondary')  !important;
+    color: map-get($adm-colors, 'primary2') !important;
+    box-shadow: none !important;
+  }
+  .v-btn__overlay {
+    opacity: 0 !important;
+  }
+}
+.v-theme--dark {
+  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat),  {
+    background-color: hsla(0, 0%, 100%, 0.35) !important;
+    color: hsla(0, 0%, 100%, 0.95) !important;
+    box-shadow: none !important;
+    opacity: 0.36;
+  }
+  .v-btn__overlay {
+    opacity: 0 !important;
+  }
+}
+
```

Comment:
The styles affects other buttons with `variant="text"`

![image](https://github.com/Adamant-im/adamant-im/assets/25831507/08bd97c1-75ce-4f24-b844-7a1e701f3809)

You must apply these styles only for `variant="elevated"` which is by default.

See all variants https://vuetifyjs.com/en/api/v-btn/#props-variant